validation/nsa/src/rules/2-noLowUserId.ts

```diff
@@ -0,0 +1,34 @@
+import { defineRule } from "@monokle/validation/custom";
+import { validatePodSpec } from "../utils.js";
+
+export const noLowUserId = defineRule({
+  id: 2,
+  description: "Disallow running with a low user ID",
+  help: "Set 'containers[].securityContext.runAsUser' to an integer > 10000.",
+  validate({ resources }, { report }) {
+    validatePodSpec(resources, (resource, pod, prefix) => {
+      const podRunAsUser = pod.securityContext?.runAsUser;
+      const podValid = podRunAsUser && podRunAsUser > 10000;
```

Comment:
We're able to detect whether the pod's security level makes this fine! With OPA we get false positives here.

Reply:
is this because these kinds of rules can't be expressed in rego? or because the OPA rule just isn't well made?

Reply:
I recall looking into it in May and couldn't get to a solution, maybe it's because it cannot be expressed but more likely it's because my understanding of the language is too limited.

Reply:
ok thanks - would be interesting to know at some point..